install.sh

```diff
@@ -31,6 +31,12 @@ download() {
   fi
 }
 
+validate_url() {
```

Comment:
Please make validate_url() work with either curl or wget.
E.g. use `curl -l`
See download().

Reply:
@umireon thanks for the suggestion 👌 I was wondering if it was necessary when doing the implementation since the doc use `curl` https://pnpm.io/installation#nodejs-is-not-preinstalled

Reply:
Support both.
macOS does not have wget by default and Debian does not have curl by default.
Universal means it works everywhere.

Reply:
definitely, I added the fallback code